dCommon/dEnums/eReponseMoveItemBetweenInventoryTypeCode.h

```diff
@@ -0,0 +1,21 @@
+#ifndef __EREPONSEMOVEITEMBETWEENINVENTORYTYPECODE__H__
+#define __EREPONSEMOVEITEMBETWEENINVENTORYTYPECODE__H__
+
+#include <cstdint>
+
+enum class eReponseMoveItemBetweenInventoryTypeCode : int32_t {
```

Comment:
any particular reason this is signed and not unsigned?

Reply:
msvc defaults to int